scripts/uberenv/spack_configs/blueos_3_ppc64le_ib_p9/packages.yaml

```diff
@@ -18,64 +18,136 @@ packages:
   all:
     compiler: [gcc, intel, pgi, clang, xl, nag]
     providers:
-      awk: [gawk]
-      blas: [openblas]
-      daal: [intel-daal]
-      elf: [elfutils]
-      golang: [gcc]
-      ipp: [intel-ipp]
-      java: [jdk]
-      lapack: [openblas]
-      mkl: [intel-mkl]
-      mpe: [mpe2]
+      blas: [netlib-lapack]
+      lapack: [netlib-lapack]
       mpi: [mvapich2]
-      opencl: [pocl]
-      openfoam: [openfoam-com, openfoam-org, foam-extend]
-      pil: [py-pillow]
-      scalapack: [netlib-scalapack]
-      szip: [libszip, libaec]
-      tbb: [intel-tbb]
-      jpeg: [libjpeg-turbo, libjpeg]
+
 # LLNL blueos CUDA
   cuda:
     paths:
-      cuda@10.1.168: /usr/tce/packages/cuda/cuda-10.1.168
+      cuda@10.1.243: /usr/tce/packages/cuda/cuda-10.1.243
```

Comment:
@bmhan12 -- will your code work with ``cuda@10.1.234`` ?  I think you said it should not be a problem.

Reply:
I asked around what codes were using very informally.